store-pg/src/main/scala/com/socrata/pg/store/PGSecondary.scala

```diff
@@ -551,6 +551,9 @@ class PGSecondary(val config: Config) extends Secondary[SoQLType, SoQLValue] wit
         loader.flush()
       }
     }
+    truthSchema.values.find(_.isSystemPrimaryKey).foreach { pkCol =>
+      sLoader.makeSystemPrimaryKey(pkCol)
+    }
```

Comment:
No, AFAICT the soql-pg-adaptor doesn't do anything special for user keys at all.